start-jut.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+WORKSPACE=/tmp JENKINS_WAR=jenkins-2.150.2.war TYPE=winstone PLUGINS_DIR=/Users/hafner/Development/jenkins/plugins ./jut-server.sh -n 1
```

Comment:
This appears to be committed by accident.

Reply:
Ups, sorry.